HHS/OPRE-OPS
# feat: adds CAN budget form

## What changed

- adds CAN budget form

## Issue

#310 #3099

## How to test

1. e2e tests pass
2. logging in as budget team
3. goto CANs details Funding Budget `/cans/504/funding`
4. select current fiscal year from drop down
5. enter edit mode via edit button
6. enter Budget Amount into input
7. click on `Add FY Budget` button
8. see value populate the card on the right `FY CAN Budget Card`
9. click on `Save Changes` button
10. see results on CAN Funding page

## Screenshots

_If relevant, e.g. for a front-end feature_

## Definition of Done Checklist
